packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -173,12 +174,11 @@ class OpPerfTelemetry {
     private recordPingTime(latency: number) {
         this.pingLatency = latency;
         // logging one in every 1000 pongs, including the first time, if it is a "write" client.
-        if (this.pongCount % 1000 === 0 && this.deltaManager.active) {
+        if (this.pongCount % 100 === 0 && this.deltaManager.active) {
```

Comment:
If the goal is to "report about every X minutes" maybe consider making this conditional based on a "lastReportedTimestamp" rather than counting pongs?